src/api/common/types.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+'use strict';
+
+export type AmountRippled = {

```

Comment:
maybe better to rename it to `RippledAmountIOU` and rename `SourceAmount` to `RippledAmount`